backend/tests/helpers/UserHelper.test.js

```diff
@@ -0,0 +1,67 @@
+const { expect } = require( 'chai' );
```

Comment:
Rename this file or rename the code file because names are different

Reply:
Already done.